chatgpt_wrapper/chatgpt.py

```diff
@@ -488,6 +392,17 @@ def new_conversation(self):
 class ChatGPT:
 
     def __init__(self, headless: bool = True, browser="firefox", model="default", timeout=60, debug_log=None, proxy: Optional[ProxySettings] = None):
+        """Create ChatGPT
+
+        Args:
+            headless (bool, optional): _description_. Defaults to True.
+            browser (str, optional): The firefox to use. Only support creating new browser. To use existing browser, use `AsyncChatGPT` instead. Defaults to "firefox".
```

Comment:
The firefox to use? Can't a user use a different browser?

Reply:
Ahh this was a typo 🤣 

But you reminded me... A bug is here too